### PR TITLE
0.9.4

### PR DESCRIPTION
**Pull request by BartoszLitwiniuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 * Gradle
 ```
 dependencies {
-    implementation 'com.github.backtrace-labs.backtrace-log4j2:backtrace-log4j2:0.9.1'
+    implementation 'com.github.backtrace-labs.backtrace-log4j2:backtrace-log4j2:0.9.4'
```
Comment (konraddysput) [inline README.md:12]: are you sure we should bump the version to 0.9.4?

File: src/main/java/backtrace/io/log4j2/Appender.java [hunk 2]
```diff
@@ -99,19 +100,31 @@ public void append(LogEvent logEvent) {
...
+    }
+
 
     /**
      * Close all of Backtrace library resources and wait until last of messages will be process and
      */
     @Override
     public void stop() {
-        internalLogger.debug("Closing BacktraceAppender");
+        internalLogger.debug("Closing BacktraceAppender..");
```
Comment (konraddysput) [inline src/main/java/backtrace/io/log4j2/Appender.java:121]: do we need additional `..` ?

**Review by konraddysput — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (BartoszLitwiniuk) [reply README.md:12]: let's keep it in the same version as backtrace-java

File: src/main/java/backtrace/io/log4j2/Appender.java (hunk 2 above)
Comment (BartoszLitwiniuk) [reply src/main/java/backtrace/io/log4j2/Appender.java]: no, I will remove it

**Pull request merged**